corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -503,6 +530,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                 sortIndices: this.options.sortIndices,
                 isMultiSelect: this.isMultiSelect,
                 selectedCaseIds: this.selectedCaseIds,
+                selectedValuesKey: this.options.queryKey,
```

Comment:
Where does `options.queryKey` come from? I don't see it being set anywhere. In the current code all usages of `queryKey` are via `sessionStorage`: `sessionStorage.queryKey`

Reply:
ah this solves some issues I was facing thank you 😊